ui/src/containers/InventoryLayout.js

```diff
@@ -140,6 +172,30 @@ const InventoryLayout = (props) => {
             </TableBody>
           </Table>
         </TableContainer>
+        <InventoryFormModal
+          title='Create'
+          formName='inventoryCreate'
+          products = {products}
+          isDialogOpen={isCreateOpen}
+          handleDialog={toggleModals}
+          handleInventory={createInventories}
+          initialValues={{}}
```

Comment:
Add default/initial values here since they override the `defaultValue` in the field components.
E.g.,:
`initialValues={{
    description: '',
    averagePrice: 0,
    amount: 0,
    bestBeforeDate: moment(new Date()).format('YYYY-MM-DD'),
    neverExpires: false,
  }}`